### PR TITLE
SAK-48451 fix criterion edit popover

### PR DESCRIPTION
**Pull request by adrianfish**
https://sakaiproject.atlassian.net/browse/SAK-48451

### CODE REVIEW

File: webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric.js [hunk 1]
```diff
@@ -224,6 +224,8 @@ export class SakaiRubric extends RubricsElement {
 
   toggleRubric(e) {
 
+    console.log("HERERE");
```
Comment (jesusmmp) [inline webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric.js:227]: Please remove this and the others 2 console.log(). Thanks!

Comment (adrianfish) [reply webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric.js:227]: Lol. Good spot :)

Comment (adrianfish) [inline webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric.js:227]:
```suggestion
```

File: webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric.js [hunk 2]
```diff
@@ -234,10 +235,12 @@ export class SakaiRubric extends RubricsElement {
     const icon = $(`#rubric_toggle_${this.rubric.id} span`);
 
     if (collapse.is(":visible")) {
+      console.log("HERE1");
```
Comment (adrianfish) [inline webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric.js:238]:
```suggestion
```

File: webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric.js [hunk 3]
```diff
@@ -238,6 +239,7 @@ export class SakaiRubric extends RubricsElement {
       titlecontainer.classList.add("active");
       icon.removeClass("fa-chevron-right").addClass("fa-chevron-down");
     } else {
+      console.log("HERE2");
```
Comment (adrianfish) [inline webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric.js:242]:
```suggestion
```

File: webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric.js [hunk 4]
```diff
@@ -224,6 +224,7 @@ export class SakaiRubric extends RubricsElement {
 
   toggleRubric(e) {
 
+
```
Comment (jesusmmp) [inline webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric.js:227]: Please, could you remove this line? thanks

File: webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric-criterion-edit.js [hunk 5]
```diff
@@ -80,91 +98,45 @@ export class SakaiRubricCriterionEdit extends RubricsElement {
...
 
   saveEdit(e) {
 
     e.stopPropagation();
 
-    const title = document.getElementById(`criterion-title-field-${this.criterion.id}`).value;
-    const description = this.criterionClone.description;
+    //const title = document.getElementById(`criterion-title-field-${this.criterion.id}`).value;
+    const title = e.target.closest(".popover-body").querySelector("input").value;
+    //const description = this.criterionClone.description;
```
Comment (jesusmmp) [inline webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric-criterion-edit.js:131]: Remove this comment and the other line 129. Thanks

Comment (adrianfish) [inline webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric-criterion-edit.js:131]:
```suggestion
    const title = e.target.closest(".popover-body").querySelector("input").value;
```

File: webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric.js (hunk 4 above)
Comment (adrianfish) [inline webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric.js:227]:
```suggestion
```

**Review by jesusmmp — APPROVED** [review]

**Comment by adrianfish** [comment]
@jesusmmp Thanks for the review :)

**Review by ern — APPROVED** [review]

**Pull request merged**